suite-native/atoms/src/Skeleton/BoxSkeleton.tsx

```diff
@@ -19,7 +26,6 @@ type BoxSkeletonProps = {
     borderRadius?: number;
 };
 
-const GRADIENT_OFFSET_DIVISOR = 8;
 const ANIMATION_DURATION = 1200;
```

Comment:
nit: This feels quite fast to me. Maybe a bit longer duration 2 or 3 seconds would feel better for animation?

Reply:
it feels quite nice for skeleton loader, we can easily increase it if needed